### PR TITLE
Ra 2020

### PR DESCRIPTION
**Pull request by fawques**

### CODE REVIEW

File: LCU.NET.csproj [hunk 1]
```diff
@@ -37,8 +38,8 @@
     <StartupObject />
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.11.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\..\..\..\..\..\..\..\Users\vguzman\Development\vasone\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
```
Comment (VoldarGames) [inline LCU.NET.csproj]: Reference to a path on your machine

File: LeagueSocket.cs [hunk 2]
```diff
@@ -113,6 +113,8 @@ private void Socket_MessageReceived(string message)
             if (IsPlaying)
                 return;
 
+            // The spellId is sometimes such a big number that the deserializer uses BigInteger, then crashes when trying to set it to int
+            message = message.Replace ("18446744073709552000", "-1").Replace ("18446744073709551615", "-1");
```
Comment (VoldarGames) [inline LeagueSocket.cs:117]: W T F

**Review by VoldarGames — CHANGES_REQUESTED** [review]

File: LCU.NET.csproj (hunk 1 above)
Comment (fawques) [reply LCU.NET.csproj]: Yeah, VS did something really weird there.
Axel already fixed it

File: LeagueSocket.cs (hunk 2 above)
Comment (fawques) [reply LeagueSocket.cs:117]: 🌈 magic 🌈 

This is adapted from LeagueClient.cs, where he did about the same with the comment `y u do dis to me`

¯\\\_(ツ)_/¯

**Review by VoldarGames — APPROVED** [review]

**Review by VoldarGames — APPROVED** [review]

**Pull request merged**